bin/nek

```diff
@@ -85,7 +85,7 @@ fi
 
 SESSION=${REAFILE%.rea}
 
-rm -f vtk/*
+#rm -f vtk/*
```

Comment:
We should think of what to do about this. If you delete it screws up the restart, but if you don't delete then Paraview gets confused if you run a simulation with less vtks...